index.html

```diff
@@ -37,190 +44,191 @@ <h2>Software Developer</h2>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li>JS</li>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Ruby</li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li>JS</li>
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>Ruby</li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+        <div class="work-container">
+          <div class="project-img"></div>
+          <div class="project-details">
+            <h2>Multi-post stories</h2>
+            <p>A daily selection of privately personalized reads;<br>
+              no accounts or sign-ups required. has been the <br>
+              industry's standard dummy text ever since the <br>
+              1500s, when an unknown printer took a standard 
+              dummy text.</p>
+            <ul class="programming-languages">
+              <li><a href="">CSS</a></li>
+              <li><a href="">html</a></li>
+              <li><a href="">bootstrap</a></li>
+              <li><a href="">Ruby</a></li>
+            </ul> <button type="button" class="btn" id="1">See project</button>
+          </div>
+        </div>
+
+      </section>
+
+
+      <section class="contact-form">
+        <h2 class="contact-us">Let’s chat!</h2>
+        <p class="paragraph3">If you have an application you are
+          interested in developing, a feature that you <br>
+          need built or a project that needs coding.
+          I'd love to help with it </p>
+          <ul>
+            <li class="full name">
+              <input type="text" id="jeremy" name="user_name" />
+           <!-- </li>
+            <li> -->
+              <input type="text" id="lastname" name="last_name" />
+            </li>
+
+            <li>
+              <input type="email" id="mail" name="user_email" />
+            </li>
+            <li>
+              <textarea id="msg" name="user_message"></textarea>
```

Comment:
As shown in the design, kindly add the corresponding placeholders to your input fields 😄. 

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199632409-dce05dac-089b-4a74-9034-b90c1141f9e8.png)  | ![image](https://user-images.githubusercontent.com/42799579/199632393-6082d008-0a30-483a-8af5-9f62fe000b7d.png)  |

You could do it by adding the `placeholder` property inside the element, like this:
```html
<input type="text" name="user_name" placeholder="Name" />
```